rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec

```diff
@@ -0,0 +1,115 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_scc_manager
+%global plugin_name scc_manager
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.0.1
+Release: 1%{?foremandist}%{?dist}
+Summary: Suse Customer Center plugin for Foreman
+Group: Applications/Systems
+License: GPL-3.0
+URL: https://www.orcharhino.com/
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+Requires: foreman >= 1.13.0
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(foreman-tasks)
+Requires: %{?scl_prefix}rubygem(katello) >= 3.2
+BuildRequires: foreman-assets
+BuildRequires: foreman-plugin >= 1.13
+BuildRequires: %{?scl_prefix}rubygem(katello) >= 3.2
+BuildRequires: %{?scl_prefix}rubygem(foreman-tasks)
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+Provides: foreman-plugin-%{plugin_name}
+
+%description
+Foreman plugin to sync SUSE Customer Center products and repositories into
+Katello.
+
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}.
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} - << \EOF}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
+
+%build
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install
+%{?scl:EOF}
```

Comment:
Unless this gem has bin files or c-extensions, you can reduce this down to using just these three lines in the prep phase similar to rubygem-katello